app/assets/javascripts-es6/item-select.js

```diff
@@ -0,0 +1,37 @@
+$(document).ready(() => {
```

Comment:
Why do you still need this?

Ideally your JS is loaded in a script tag that comes at the very bottom of the `<body>` tag, so you can safely assume that the DOM is ready by the time the JS is executed.

Would be nice to have zero jQuery!

Reply:
I tried moving the rails helper tag for the asset pipeline down to the end of the body tag, and taking out the `$(document).ready({})` out of the js. It looks okay when I look at the page source, but the code doesn't seem to be running. Once I understand more of how rails actually works under the hood, hopefully I can figure out why. So I'm leaving it in for now....Thanks man!!!